g2core/board/ArduinoDue/gShield-pinout.h

```diff
@@ -258,12 +258,12 @@ pin_number kADC14_PinNumber = -1;  // Not physially pinned out
 
 
 // GRBL / gShield compatibility pins -- Due board ONLY
-
+// NOTE: These dont appear to work
```

Comment:
I can do that, however INPUT_ACTION_CYCLE_START and INPUT_ACTION_RESET aren't implemented (afaik) and my attempts to make them work were not successful. INPUT_ACTION_HOLD is weird because I cant figure out how to unhold...